app/controllers/quizzes_controller.rb

```diff
@@ -0,0 +1,20 @@
+class QuizzesController < ApplicationController
```

Comment:
let's keep the same actions order as in scaffolded controllers:

1. index
2. show
3. new
4. edit
5. create
6. update
7. destroy